toqito/state_props/has_symmetric_extension.py

```diff
@@ -134,7 +134,7 @@ def has_symmetric_extension(
     # If the optimal value of the symmetric extension hierarchy is equal to 1,
     # this indicates that there does not exist a symmetric extension at
     # level :code:`level`.
-    return not np.isclose(
+    not np.isclose(
         (1 - min(symmetric_extension_hierarchy([rho], probs=None, level=level), 1)),
         0,
         atol=tol,
```

Comment:
Did you mean to make this change?

Reply:
ye, I will undo that, because at first I thought it cause some testcase error